companion/src/generaledit.cpp

```diff
@@ -13,53 +13,43 @@
 #define BEEP_VAL_SHIFT   3
 
 GeneralEdit::GeneralEdit(RadioData &radioData, QWidget *parent) :
-    QDialog(parent),
```

Comment:
Please next time, try not to reindent in the same commit than modifications, it's easier to understand what changed!